salt/cloud/clouds/msazure.py

```diff
@@ -59,9 +59,12 @@
 try:
     import azure
     import azure.servicemanagement
+    from azure.storage import BlobService
     from azure import (WindowsAzureConflictError,
                        WindowsAzureMissingResourceError,
                        WindowsAzureError)
+    import salt.utils.msazure
+    from salt.utils.msazure import object_to_dict
```

Comment:
perhaps, instead of 2 imports ....
```python
import salt.utils.msazure as msazure
```

and then you can use it like:

```python
msazure.object_to_dict()
# and/or
msazure.foo_bar_whatever
```